docs/src/prototypes/Prototypes.tsx

```diff
@@ -3,27 +3,34 @@ import { Box, Header, Segment } from '@stardust-ui/react'
 
 interface PrototypeSectionProps {
   title?: React.ReactNode
+  style?: React.CSSProperties
 }
 
 interface ComponentPrototypeProps extends PrototypeSectionProps {
   description?: React.ReactNode
 }
 
-export const PrototypeSection: React.FC<ComponentPrototypeProps> = props => (
-  <Box style={{ margin: 20 }}>
-    {props.title && <Header as="h1">{props.title}</Header>}
-    {props.children}
-  </Box>
-)
+export const PrototypeSection: React.FC<ComponentPrototypeProps> = props => {
+  const { title: title, children, style, ...rest } = props
+  return (
+    <Box style={{ margin: 20, ...style }} {...rest}>
```

Comment:
Just curious, why we use style here instead of styles?

Reply:
we implemented this component when `styles` was not a popular prop at all (we were advising users to avoid it). Changed to `styles`, thanks!